### PR TITLE
Editted formatting of README.txt in sunpy/changelog,sunpy/docs,sunpy/…

### PR DESCRIPTION
**Pull request by Mmendero**
…examples,sunpy/requirements

### Description

Fixes #<Issue Number>

### CODE REVIEW

File: examples/README.txt [hunk 1]
```diff
@@ -1,7 +1,5 @@
-.. _examples:
```
Comment (nabobalis) [inline examples/README.txt]: This needs to be left in.

File: changelog/README.rst [hunk 2]
```diff
@@ -1,3 +1,7 @@
+Changelog
+=========
+
+
```
Comment (nabobalis) [inline changelog/README.rst]: Why two new lines?

**Comment by dpshelio** [comment]
invalid?

**Comment by nabobalis** [comment]
@dpshelio Only if we get no more traction on this. I think the changes actually need to be done. Our headers for our readmes are not correct. But it needs more before we can merge. IF not, I plan to do it myself in a different PR.

**Comment by nabobalis** [comment]
I'm going to close this. I have a WIP branch that has this commit in its history but I' ve made the changes and others upon of it. Please feel free to tackle other issues!

**Pull request closed without merging**